corehq/apps/hqwebapp/widgets.py

```diff
@@ -149,15 +151,20 @@ def __init__(self, choices=None, attrs=None, apply_bindings=False):
     def render(self, name, value, attrs=None, renderer=None):
         return '''
             <select-toggle data-apply-bindings="{apply_bindings}"
-                           params="name: '{name}',
-                                   id: '{id}',
-                                   value: {value},
-                                   options: {options}"></select-toggle>
-        '''.format(apply_bindings="true" if self.apply_bindings else "false",
-                   name=name,
-                   id=html_attr(attrs.get('id', '')),
-                   value=html_attr(self.params['value'] or '"{}"'.format(html_attr(value))),
-                   options=html_attr(json.dumps([{'id': c[0], 'text': c[1]} for c in self.choices])))
+                            params="name: '{name}',
+                                    id: '{id}',
+                                    value: {value},
+                                    options: {options}"></select-toggle>
+        '''.format(
+            apply_bindings="true" if self.apply_bindings else "false",
+            name=name,
+            id=html_attr(attrs.get('id', '')),
+            value=html_attr(self.params['value'] or '"{}"'.format(html_attr(value))),
+            options=html_attr(json.dumps(
+                [{'id': c[0], 'text': c[1]} for c in self.choices],
```

Comment:
Looking at this again more closely, I realized you can also just coerce the text to a string directly. In more complicated JSON structures, that's not always practical.  I don't see a big advantage to one vs the other, just wanted to mention it.
```suggestion
                [{'id': c[0], 'text': str(c[1])} for c in self.choices],
```

Reply:
Cool!